Makefile

```diff
@@ -321,11 +324,28 @@ load-addon-operator: build-image-addon-operator-manager
 			--name=$(KIND_CLUSTER_NAME);
 .PHONY: load-addon-operator
 
-# Template deployment
+## Load Addon Operator Webhook images into kind
+load-addon-operator-webhook: build-image-addon-operator-webhook
+	@source hack/determine-container-runtime.sh; \
+		$$KIND_COMMAND load image-archive \
+			.cache/image/addon-operator-webhook.tar \
+			--name=$(KIND_CLUSTER_NAME);
+.PHONY: load-addon-operator-webhook
+
+# Template deployment for Addon Operator
 config/deploy/deployment.yaml: FORCE $(YQ)
 	@yq eval '.spec.template.spec.containers[0].image = "$(ADDON_OPERATOR_MANAGER_IMAGE)"' \
 		config/deploy/deployment.yaml.tpl > config/deploy/deployment.yaml
 
+# Template deployment for Addon Operator Webhook
+config/deploy/webhook/deployment.yaml: FORCE $(YQ)
+	@yq eval '.spec.template.spec.containers[0].image = "$(ADDON_OPERATOR_WEBHOOK_IMAGE)" | \
```

Comment:
the yq command string can't have a newline inside of it (at least with make on linux machines)